src/styles/_card.scss

```diff
@@ -0,0 +1,83 @@
+.card {
+  width: 200px;
+  padding: 10px;
+  border-radius: 8px;
+  cursor: pointer;
+
+  @include mq('desktop', $breakpoints) {
+    display: flex;
+    width: 340px;
+  }
+
+  &:hover {
+    background: #fff;
```

Comment:
Ustawiając `background` warto zawsze ustawiać też `color` by nie dopuścić do sytuacji, że `color` zostanie odziedziczony taki sam jak nasz ustawiony `background` - przez to tekst będzie niewidoczny.